runners/core-java/src/main/java/org/apache/beam/runners/core/helpers/package-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Provides helpers for Beam runner authors. */
+@DefaultAnnotation(NonNull.class)
+package org.apache.beam.runners.core.helpers;
```

Comment:
I think we can put `Concatenate` directly into `org.apache.beam.runners.core`. `org.apache.beam.runners.core` already serves the same purpose as the proposed `helpers` subpackage.

https://github.com/apache/beam/blob/3a7b8e757d0c220b5838ca6a0805ee463b24f85d/runners/core-java/src/main/java/org/apache/beam/runners/core/package-info.java#L19

Reply:
agree, done